corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -292,7 +292,7 @@ def remove_residual_indices(self):
         """
         Remove the residual indices that are not used by HQ
         """
-        existing_indices = es_manager.get_indices()
+        existing_indices = es_manager.indices_info()
```

Comment:
nit: based solely on names, it isn't very clear why one of these would return closed indices but the other would not. Perhaps clearer names could help?

Reply:
Ahh! Yeah. Actually `indices_info` and `get_indices` both call differnent ES APIs to get the data. 
I can't think of what names would be better here, given that both of them return different info related to the indices. I can add this details to the documentation of these functions but if you have some name suggestions, I would happily update them here.

Reply:
https://github.com/dimagi/commcare-hq/pull/33491/commits/c6b7376106812fa18e151cf20577fd35c165bcc9